From 5d0e9b6a4669ff0d72f6d646b8e3380246b8afcc Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Fri, 05 Oct 2018 16:15:18 +0300
Subject: [PATCH] flah_burn: Update return value

There are two places with exit 3
Turn one to exit 4,
The one which do not match KMD API.

TODO: Update KMD API (enum) to support it (burn failure)
---

diff --git a/board/habanalabs/common/rootfs_overlay/usr/sbin/flash_burn b/board/habanalabs/common/rootfs_overlay/usr/sbin/flash_burn
index 7e7a0ab..480977b 100755
--- a/board/habanalabs/common/rootfs_overlay/usr/sbin/flash_burn
+++ b/board/habanalabs/common/rootfs_overlay/usr/sbin/flash_burn
@@ -101,7 +101,7 @@
 
 	if [ $? -ne 0 ]; then
 		echo "ERROR: flashcp ${IMG} ${DEV}"
-		exit 3
+		exit 4
 	fi
 
 	IMG_SIZE=`ls -l ${IMG}						| tr -s ' ' 			| cut -d' ' -f5`
